README.md

```diff
@@ -26,6 +26,10 @@ Install using pip
 
 Run `curator --help` for usage specifics.
 
+### Common Issues
```

Comment:
And you could link that entire header to http://www.elastic.co/guide/en/elasticsearch/client/curator/current/faq.html